pages/api/avcs/index.ts

```diff
@@ -0,0 +1,115 @@
+/*
+
+SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+
+*/
+
+import validateQueryParam from 'modules/app/api/validateQueryParam';
+import withApiHandler from 'modules/app/api/withApiHandler';
+import { fetchAvcs } from 'modules/avcs/api/fetchAvcs';
+import { AvcOrderByEnum } from 'modules/avcs/avcs.constants';
+import { AvcsAPIResponse } from 'modules/avcs/types/avc';
+import { DEFAULT_NETWORK, SupportedNetworks } from 'modules/web3/constants/networks';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+/**
+ * @swagger
+ * paths:
+ *  /api/avcs:
+ *    get:
+ *      tags:
+ *        - "AVCs"
```

Comment:
I'm not sure why but I don't see this show up on the API docs page as it's own section. I thought adding the tag of "AVCs" would have done that.

Reply:
Apparently it was an issue with the last line where `$ref` needed to be on a new line. Fixed now.